src/Snippets/Docs/CircuitBreaker.cs

```diff
@@ -0,0 +1,83 @@
+﻿using System.Net;
+using System.Net.Http;
+using Polly;
+using Polly.CircuitBreaker;
+using Snippets.Docs.Utils;
+
+namespace Snippets.Docs;
+
+#pragma warning disable S103 // Lines should not be too long
+
+internal static class CircuitBreaker
+{
+    public static async Task Usage()
+    {
+        #region circuit-breaker
+
+        // Use the default Circuit Breaker options with pre-configured settings:
+        //
+        // FailureRatio: 0.1 — Opens the circuit if 10% of the calls fail
+        // BreakDuration: 30 seconds — Duration the circuit stays open before another try
+        // MinimumThroughput: 100 — Minimum number of calls required before the circuit can break
+        // SamplingDuration: 30 seconds — Time window for monitoring calls before allowing the circuit to break
+        // ShouldHandle: Handles all exceptions except for OperationCanceledException
+        new ResiliencePipelineBuilder().AddCircuitBreaker(new CircuitBreakerStrategyOptions());
+
+        // Configure custom settings for the Circuit Breaker:
+        //
+        // The circuit will break if more than 50% of actions result in handled exceptions,
```

Comment:
To be consistent with the words used further up.

```suggestion
        // The circuit will break if more than 50% of calls result in handled exceptions,
```